File: app/src/main/java/com/motrixi/datacollection/utils/UploadLogUtil.java
package com.motrixi.datacollection.utils;

import android.provider.Settings;

import com.adobe.fre.FREContext;
import com.motrixi.datacollection.content.Contants;

/**
 * author : Jason
 * date   : 2020/12/8 12:33 PM
 * desc   :
 */
public class UploadLogUtil {

    public static void uploadLogData(FREContext context, String value) {

        String val = MessageUtil.uploadLog(value);
        String appKey = Contants.APP_KEY;
        String androidID = Settings.System.getString(context.getActivity().getContentResolver(), Settings.Secure.ANDROID_ID);

        /*Call<JsonObject> call = HttpClient.uploadLog(context, val, appKey, androidID);
        call.enqueue(new Callback<JsonObject>() {
            @Override
            public void onResponse(Call<JsonObject> call, Response<JsonObject> response) {

            }

            @Override
            public void onFailure(Call<JsonObject> call, Throwable t) {

            }

        });*/
    }
}

File: app/src/main/java/com/motrixi/datacollection/utils/UploadLogUtil.kt
package com.motrixi.datacollection.utils

import android.content.Context
import android.provider.Settings
import android.util.Log
import com.google.gson.JsonObject
import com.motrixi.datacollection.content.Contants
import com.motrixi.datacollection.content.Session
import com.motrixi.datacollection.network.HttpClient
import org.json.JSONObject
import retrofit2.Callback
import retrofit2.Response

/**
 * author : Jason
 *  date   : 2020/11/18 1:23 PM
 *  desc   :
 */
object UploadLogUtil {

    fun uploadLogData(context: Context, value: String) {

        var value = MessageUtil.uploadLog(value)
        var appKey = Contants.APP_KEY
        var androidID = Settings.System.getString(context.contentResolver, Settings.Secure.ANDROID_ID)

        var call = HttpClient.uploadLog(context, value, appKey, androidID)
        call.enqueue(object : Callback<JsonObject> {
            override fun onFailure(call: retrofit2.Call<JsonObject>, t: Throwable) {

            }

            override fun onResponse(call: retrofit2.Call<JsonObject>, response: Response<JsonObject>) {

            }
        })
    }
}